test/core/client_channel/lb_policy/lb_policy_test_lib.h

```diff
@@ -375,16 +385,6 @@ class LoadBalancingPolicyTest : public ::testing::Test {
   LoadBalancingPolicyTest()
       : work_serializer_(std::make_shared<WorkSerializer>()) {}
 
-  void TearDown() override {
-    // Note: Can't safely trigger this from inside the FakeHelper dtor,
-    // because if there is a picker in the queue that is holding a ref
-    // to the LB policy, that will prevent the LB policy from being
-    // destroyed, and therefore the FakeHelper will not be destroyed.
-    // (This will cause an ASAN failure, but it will not display the
-    // queued events, so the failure will be harder to diagnose.)
-    helper_->ExpectQueueEmpty();
```

Comment:
I think the right solution here is just to check whether `helper_` is null here before calling a method on it.